chaco/tests/test_arraydatasource.py

```diff
@@ -247,7 +254,7 @@ def test_serialization_state_no_persist(self):
             "_min_index",
             "_max_index",
         ]:
-            self.assertIn(key, state)
+            self.assertNotIn(key, state)
```

Comment:
If not persisting they should not be in state, but if we are they should be. I believe these tests are doing what they are supposed to now